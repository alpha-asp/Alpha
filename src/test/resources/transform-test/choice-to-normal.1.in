{ red(N); green(N); blue(N) } :- node(N).