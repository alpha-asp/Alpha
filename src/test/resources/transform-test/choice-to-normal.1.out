_nred("1", N) :- node(N), not red(N).
red(N) :- node(N), not _nred("1", N).
:- node(N), red(N), _nred("1", N).
_ngreen("1", N) :- node(N), not green(N).
green(N) :- node(N), not _ngreen("1", N).
:- node(N), green(N), _ngreen("1", N).
_nblue("1", N) :- node(N), not blue(N).
blue(N) :- node(N), not _nblue("1", N).
:- node(N), blue(N), _nblue("1", N).
